crates/ln-dlc-node/src/seed.rs

```diff
@@ -162,4 +177,17 @@ mod tests {
             "1cf21ab62bf5a5ee40896158cbbc18b9ad75805e1824a252d8060c6c075b228f"
         );
     }
+
+    #[test]
+    fn restore_same_seed_from_exported_mnemonic() {
+        let seed = Bip39Seed::new().unwrap();
+        let seed_words = seed.get_seed_phrase().join(" ");
+
+        let restore_path = &temp_dir().join("seed_restored");
+        let seed_restored = Bip39Seed::restore_from_mnemonic(&seed_words, restore_path).unwrap();
+
+        assert!(seed ==  seed_restored, "Restored seed should be the same as the original seed");
```

Comment:
Any reason why you didn't go with `assert_eq!`?

Reply:
yes, `assert_eq` would need me to derive `Debug`, which in turn would make our mnemonic trivially printable. I would prefer to avoid that.
I'll add a comment to that extent.